chef/data_bags/crowbar/bc-template-cinder.json

```diff
@@ -19,6 +19,9 @@
       "use_gitbarclamp": true,
       "use_virtualenv": true,
       "use_pip_cache": true,
+      "min_pool_size": 10,
+      "max_pool_size": 100,
+      "pool_timeout": 2,
```

Comment:
@rossella same values as neutron sound good to me.

I'll create a trello card for nova, thanks for confirming this!